Content.Server/GameTicking/Rules/Configurations/GenericGameRuleConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using JetBrains.Annotations;
+
+namespace Content.Server.GameTicking.Rules.Configurations;
+
+/// <summary>
+/// A generic configuration, for game rules that don't have special config data.
+/// </summary>
+[UsedImplicitly]
+public sealed class GenericGameRuleConfiguration : GameRuleConfiguration
+{
+    [DataField("id", required: true)]
+    private string _id = default!;
+    public override string Id => _id;
```

Comment:
This is uhh. Is this because of the string overrides?

Reply:
Yea. I could not figure out a different way to do it (as the specialized configs hardcode in which gamerule they configure), and didn't want to pre-emptively make a config class for every single rule.